compose/foundation/foundation/build.gradle

```diff
@@ -137,7 +137,11 @@ if (AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             jsNativeMain.dependsOn(skikoMain)
             nativeMain.dependsOn(jsNativeMain)
             jsWasmMain.dependsOn(jsNativeMain)
-
+            uikitMain {
+                dependencies {
+                    implementation(project(":compose:ui:ui-uikit"))
```

Comment:
Unrelated to this CI, but `foundation` is supposed to use only stable APIs from `ui`, so it shouldn't have such dependencies

Reply:
This policy is only for `@InternalComposeUiApi`, if it is public and not annotated, it is considered stable, and we should maintain compatibility for it, even if the lib itself is internal.

Reply:
Kinda. But APIs from that Obj-C module are not supposed to be stable at the moment

Reply:
Then I see 3 choices:
- make it stable
- don't use it in foundation
- align ui/foundation versions (much bigger task with discussions)

Reply:
Also
- inlining/repackaging the code (very big task)

Reply:
> don't use it in foundation

It's the right way to go, but it requires fixing architecture layering issues that are here because of common code